inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/StreamSourceEntity.java

```diff
@@ -36,9 +36,10 @@ public class StreamSourceEntity implements Serializable {
     private String sourceName;
     private String agentIp;
     private String uuid;
+    private String subTaskAgentIps;
```

Comment:
Excuse me, what is the usage of this field?

Reply:
This field records all the sub task agent ip from the same cluster name task.

Reply:
Redudant, should be  deleted.